greenback-kit-core/src/main/java/com/greenback/kit/model/AutoExportAccount.java

```diff
@@ -0,0 +1,45 @@
+package com.greenback.kit.model;
```

Comment:
?Wouldn't this entire class be deleted since it should no longer be in our public API?